syntax = "proto3";

message idata {
    string oid = 1;
    string data = 2;
    bool stemming = 3;
    bool named_entities = 4;
    bool remove_stops = 5;
    bool lemmatize = 6;
}

message odata{
    string oid = 1;
    string data = 2;
}
service NlpProcessor {
    rpc process_nlp(idata) returns (odata) {}
}